app/server/prisma/schema.prisma

```diff
@@ -132,11 +132,11 @@ model EventQuestion {
     question String @db.VarChar(500)
     // if position is -1, then the question is not queued
     position Int @default(-1)
-    isVisible Boolean
-    isAsked Boolean
+    isVisible Boolean @default(true)
+    isAsked Boolean @default(false)
     lang String @db.VarChar(30)
-    isFollowUp Boolean
-    isQuote Boolean
+    isFollowUp Boolean @default(false)
+    isQuote Boolean @default(false)
```

Comment:
Hmm, do adding default values make this less testable?  I'd maybe hold off on adding these until we figure out testing a bit more.  If it removes some testability, I don't think we want them since they're mostly convenience.

Also, it now requires a database migration if we want to change them.  Not sure that this is the best method.  (I know I made the position default of -1, but in retrospect, I'm not 100% sure I should have).

Reply:
Alright I'll remove that commit.